AdblockPlusSafari/Info.plist

```diff
@@ -19,9 +19,20 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.1.0</string>
+	<string>1.2.1</string>
```

Comment:
Why change the version string? The last version we released was `1.0.1`, so the upcoming release should be `1.1.0`.